build.sbt

```diff
@@ -105,18 +105,23 @@ lazy val validation = (project in file("validation"))
 
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
+  .settings(Revolver.settings:_*)
   .settings(
-    assemblyMergeStrategy in assembly := {
-      case "application.conf" => MergeStrategy.concat
-      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
-      case x =>
-        val oldStrategy = (assemblyMergeStrategy in assembly).value
-        oldStrategy(x)
-    },
-    libraryDependencies ++= akkaPersistenceDeps ++ httpDeps
+    Seq(
+      assemblyJarName in assembly := {s"${name.value}.jar"},
```

Comment:
Are you sure you were in the panel project when running `assembly`?  This is what I saw:
```
[info] Packaging /Users/grippinn/HMDA/hmda-platform/panel/target/scala-2.12/panel.jar ...
[info] Done packaging.
[success] Total time: 122 s, completed Jun 22, 2017 3:50:32 PM
```